### PR TITLE
add pyode, open dynamics engine

### PR DESCRIPTION
**Pull request by repagh**
Checklist

- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ x] Source is from official source
- [ x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ x] If static libraries are linked in, the license of the static library is packaged.
- [ x] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyode/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  requires:
+    - python {{ PY_VER }}
+  imports:
+    - ode
+
+about:
+  home: https://www.ode.org/
+  license: LGPL-2.0 or BSD
+  summary: Open Dynamics Engine, dynamics simulation
+  license_file: LICENSE.TXT
```
Comment (marcelotrevisani) [inline recipes/pyode/meta.yaml]: I think it should be 
```yaml
  license_file:
    - LICENSE.TXT
    - LICENSE-BSD.TXT

File: recipes/pyode/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  number: '0'
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - expat
+    - python {{ PY_VER }}
```
Comment (marcelotrevisani) [inline recipes/pyode/meta.yaml]: What are your intention with this jinja variable?

File: recipes/pyode/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/odedevs/ode/downloads/ode-{{ version }}.tar.gz
+  sha256: 4ba3b76f9c1314160de483b3db92b0569242a07452cbb25b368e75deb3cabf27
+
+build:
+  number: '0'
```
Comment (marcelotrevisani) [inline recipes/pyode/meta.yaml]:
```suggestion
  number: 0
```

File: recipes/pyode/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/pyode/meta.yaml]: Python should be jus in ``host`` as well

File: recipes/pyode/build.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# part 1, library, cmake and c++
+mkdir _build
+cd _build
+
+cmake -DODE_WITH_DEMOS:BOOL=OFF \
+      -DODE_WITH_TESTS:BOOL=OFF .. \
+      -DCMAKE_INSTALL_PREFIX:PATH=""
```
Comment (marcelotrevisani) [inline recipes/pyode/build.sh]:
```suggestion
      -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX \
      -DCMAKE_PREFIX_PATH=$PREFIX
```

File: recipes/pyode/meta.yaml (hunk 1 above)
Comment (repagh) [reply recipes/pyode/meta.yaml]: thnkx

File: recipes/pyode/meta.yaml (hunk 2 above)
Comment (repagh) [reply recipes/pyode/meta.yaml]: OK, removed it, has no effect.

File: recipes/pyode/meta.yaml (hunk 3 above)
Comment (repagh) [reply recipes/pyode/meta.yaml]: fixed

File: recipes/pyode/build.sh (hunk 4 above)
Comment (repagh) [reply recipes/pyode/build.sh]: This failed when I tried it (& removed the DESTDIR). Looks purer my way; install under basically /lib and /include, and in the PREFIX location

**Comment by repagh** [comment]
Superseded by combined pyode and libode

**Pull request closed without merging**